src/table/utils/handle-key-press.js

```diff
@@ -164,6 +165,7 @@ export const bodyHandleKeyPress = ({
   switch (evt.key) {
     case 'ArrowUp':
     case 'ArrowDown': {
+      evt.key === 'ArrowUp' && handleNavigateTop([cell.rawRowIdx, cell.rawColIdx], rootElement);
```

Comment:
It is the current cell and not the next cell we should look at?

Reply:
it is the coordinate for the next cell when totals is top and the current cell when totals is bottom and noTotals

Reply:
as you see in the video, I think it has a better outcome.

Reply:
but wouldn't it behave a bit differently when you have totals on top or not?

Reply:
yes, I also thought it should be different but it is not. I am looking into it.

Reply:
do not really know why. I updated the current video.

Reply:
Now I understand it and it is correct.